source/indexes/atlas-search-index.txt

```diff
@@ -55,21 +80,17 @@ each Atlas Search index management method.
 Create a Search Index
 ---------------------
 
-To create a single Atlas Search Index on a collection, call the ``create_one()`` method on a ``mongocxx::search_index_view`` instance and pass in a ``mongoxcc::search_index_model`` 
-instance specifying the Search index you want to create. 
+To create a single Atlas Search index on a collection, call the ``create_one()`` method on a ``mongocxx::search_index_view`` instance and pass in a ``mongoxcc::search_index_model`` 
+instance specifying the index that you want to create. 
 
 A ``mongocxx::search_index_model`` instance requires the following fields:
 
-- ``name``, a string specifying the name for your search index
-- ``definition``, a document containing the ``mappings`` field, which specifies how how to configure fields in your search index
+- ``name``: String specifying the name for your index
+- ``definition``: Document containing the ``mappings`` field, which specifies how how to configure fields in your index
 
 .. tip:: 
 
-   For a full list of fields you can configure in an Atlas Search index, see the :atlas:`Review Atlas Search Index Syntax </atlas-search/index-definitions/>` guide in the Atlas Search Documentation.
-
-After you create a Search index, you can perform Atlas Search queries on
-your collection. To learn more, see :atlas:`Create and Run Atlas Search
-Queries </atlas-search/searching/>` in the Atlas Search Documentation.
+   For a full list of fields you can configure in an Atlas Search index, see the :atlas:`Review Atlas Search Index Syntax </atlas-search/index-definitions/>` guide in the Atlas Search documentation.
```

Comment:
I: [restrictive clauses](https://www.mongodb.com/docs/meta/style-guide/writing/use-that-which-correctly/#std-label-restrictive-clauses:~:text=Make%20sure%20to%20clarify%20restrictive%20clauses%2C%20as%20follows%3A)

```suggestion
   For a full list of fields that you can configure in an Atlas Search index, see the :atlas:`Review Atlas Search Index Syntax </atlas-search/index-definitions/>` guide in the Atlas Search documentation.
```